debugger/src/main/kotlin/org/rust/debugger/lang/RsMSVCTypeNameDecoratorVisitor.kt

```diff
@@ -31,15 +36,41 @@ class RsMSVCTypeNameDecoratorVisitor : RsTypeNameBaseVisitor<Unit>() {
     }
 
     // ptr_const$<MyType> -> *const MyType
+    // ptr_const$<str> -> *const str
+    // ptr_const$<slice$<MyType> > -> *const [MyType]
     override fun visitMsvcPtr_const(ctx: MsvcPtr_constContext) {
         builder.append("*const ")
-        ctx.type.accept(this)
+
+        // BACKCOMPAT: Rust 1.66. Get rid of if-else, replace with `ctx.type.accept(this)`
+        @Suppress("DuplicatedCode")
+        if (ctx.type.msvcTypeName()?.msvcStr() != null) {
+            builder.append("str")
+        } else if (ctx.type.msvcTypeName()?.msvcSlice() != null) {
+            builder.append("[")
+            ctx.type.msvcTypeName().msvcSlice().type.accept(this)
+            builder.append("]")
```

Comment:
Maybe extract `ctx.type.msvcTypeName()` into variable since we use them 3 times?